packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -57,20 +57,33 @@ type ChatMessageComponentAsMessageBubbleProps = {
    */
   onRenderAvatar?: OnRenderAvatarCallback;
 
-  /* @conditional-compile-remove(dateTimeCustomization) */
+  /* @conditional-compile-remove(date-time-customization) */
```

Comment:
This can still be added unconditionally (not part of public API).

I'll make your code simpler by needing less conditional compilation.